src/classes/ALLO_Allocations_TDTM.cls

```diff
@@ -45,6 +45,14 @@ public class ALLO_Allocations_TDTM extends TDTM_Runnable {
     /** @description Allocations settings. */
     public static Allocations_Settings__c settings = UTIL_CustomSettingsFacade.getAllocationsSettings();
 
+    /** @description  Retrieve and cache the household settings the first time this class is loaded only */
+    public static final npo02__Households_Settings__c householdSettings = UTIL_CustomSettingsFacade.getHouseholdsSettings();
```

Comment:
UTIL_CustomSettingsFacade already caches, do you think this extra reference is useful?

Reply:
Isn't this how we currently retrieve settings for use within a class, or is just done as `UTIL_CustomSettingsFacade.getHouseholdsSettings().Some_Field_Value__c`?

Reply:
Yep, you can reference it directly.

Reply:
For consistency, should the class be updated to use that method for 'householdSettings' as well as the `settings` (AllocationSettings) from above? Also, I find it more readable to have a static var such as householdSettings referenced in the code. If CustomSettingsFacade is already caching the data, then I think this static var is a by reference version of that Sobject.